package.json

```diff
@@ -20,25 +20,25 @@
         ]
     },
     "scripts": {
-        "_______ Hooks _______": "Yarn hooks.",
+        "_______ Hooks _______": "echo 'Yarn hooks.'",
         "postinstall": "yarn run patch && yarn lerna run patch",
-        "_______ Library Scripts #####": "Some libraries have their own build scripts.",
+        "_______ Library Scripts #####": "echo 'Some libraries have their own build scripts.'",
         "build:libs": "lerna run build:lib",
         "suite:build:web": "yarn workspace @trezor/suite-web build",
-        "_______ Start Scripts _______": "Here are standalone scripts for running individual applications for development.",
+        "_______ Start Scripts _______": "echo 'Here are standalone scripts for running individual applications for development.'",
         "suite:dev": "yarn workspace @trezor/suite-web dev",
         "suite:dev:ios": "yarn workspace @trezor/suite-native dev:ios",
         "suite:dev:android": "yarn workspace @trezor/suite-native dev:android",
         "suite:dev:desktop": "yarn workspace @trezor/suite-desktop dev",
-        "_______ Testing _______": "Testing, linting, type checking...",
+        "_______ Testing _______": "echo 'Testing, linting, type checking...'",
         "type-check": "yarn lerna run type-check",
-        "type-check:force": "rimraf -rf **/libDev && yarn type-check",
+        "type-check:force": "rimraf -rf -- **/libDev && yarn type-check",
         "test:unit": "yarn lerna run test:unit",
         "lint:js": "eslint --report-unused-disable-directives --cache --ignore-path .gitignore --ext .js,.ts,.tsx ./",
         "lint": "yarn lerna run lint:styles && yarn lint:js",
         "lint-staged": "npx lint-staged",
         "lint:shellcheck": "./scripts/shellcheck.sh",
-        "_______ Commands _______": "Useful commands and scripts.",
+        "_______ Commands _______": "echo 'Useful commands and scripts.'",
```

Comment:
the shellcheck wrapper is testing all bash scripts and also these package.json inline scripts. 
In order to check them all, they all have to be valid shell scripts, hope it is fine to wrap them in 'echo' :))